### PR TITLE
Add tab key support

### PR DESCRIPTION
**Pull request by aloscha**

### CODE REVIEW

File: juicy-checkbox-group-item.html [hunk 1]
```diff
@@ -161,7 +161,7 @@
             </template>
             <template is="dom-if" if="{{!checkboxHidden}}">
                 <div id="checkboxItemContainer">
-                    <div id="checkboxItem" data-checkable$="{{checkable}}" data-checked$="{{checked}}" on-click="toggleCheckbox">
+                    <div id="checkboxItem" data-checkable$="{{checkable}}" data-checked$="{{checked}}" on-click="toggleCheckbox" on-keypress="keypressCheckbox" tabindex="0">
```
Comment (warpech) [inline juicy-checkbox-group-item.html]: Normally such things are done with `change` event. Have you tried that?

We even do it in KitchenSink: https://github.com/StarcounterApps/KitchenSink/blob/master/src/KitchenSink/wwwroot/KitchenSink/CheckboxPage.html#L7

**Review by warpech — CHANGES_REQUESTED** [review]

**Comment by warpech** [comment]
ping @aloscha, @mmnosek, where are we with this?

File: juicy-checkbox-group-item.html (hunk 1 above)
Comment (aloscha) [reply juicy-checkbox-group-item.html]: Difference between KichetnSink and JuicyCheckboxItem is that in KitchenSink we have `input`, in juicy we have `div`

Comment (warpech) [reply juicy-checkbox-group-item.html]: Ah, I did not expect that.

Anyway, `keypress` is a strange event. See: https://www.kirupa.com/html5/keyboard_events_in_javascript.htm

Please check if you could use `keydown` or `keyup`, that would be preferable.

Comment (aloscha) [reply juicy-checkbox-group-item.html]: I think `keyup` will be enough. Thanks

**Comment by mmnosek** [comment]
@warpech I've changed keypress to keydown as you asked for.

**Review by mmnosek — APPROVED** [review]

**Review by warpech — APPROVED** [review]
Looks good! @mmnosek will you merge?

**Pull request merged**